### DISCUSSION TITLE
Speeding up loading users

### DISCUSSION THREAD

**Discussion by rqi14** [goauthentik/authentik#7077] [Q&A]
Hi. I found listing users in authentik.FQDN/if/admin/#/identity/users is extremely slow. With 10 ish users, it took several minutes to load. I'm deploying authentik using docker compose following the tutorial. Is there anyway to speed it up?

**Comment by ser** [comment]
what are parameters of your hardware? for me it takes 1 sec

**Comment by Tealk** [comment]
Since a few updates it has also been extremely slow for me, i.e. the entire web interface. The structure also looks completely different, everything is no longer in its place:

![image](https://github.com/user-attachments/assets/a58e50a7-cfcf-49d2-a095-abe0def94b6a)

![image](https://github.com/user-attachments/assets/8921d210-4e87-43eb-93c4-44d397377bba)


Specs:
```
inxi
CPU: quad core Common KVM (-MCP-) speed: 3394 MHz Kernel: 6.1.0-25-amd64 x86_64 Up: 47d 18h 23m
Mem: 1387.8/1966.5 MiB (70.6%) Storage: 100 GiB (46.1% used) Procs: 128 Shell: Bash inxi: 3.3.26
```

**Reply by Tealk** [reply]
I have now been able to determine that the problem only occurs with Firefox, Chromium-based browsers do not have this.
